routes/web.php

```diff
@@ -64,6 +64,8 @@
     Route::get('/shelves/{slug}', [BookshelfController::class, 'show']);
     Route::put('/shelves/{slug}', [BookshelfController::class, 'update']);
     Route::delete('/shelves/{slug}', [BookshelfController::class, 'destroy']);
+    Route::get('/shelves/{slug}/sync', [BookshelfController::class, 'syncWithTitan']);
```

Comment:
Please create a command for this function rather then web route

Reply:
We can give restricted access to admin only, you can give button for this route

Reply:
@chintandave-simformsolutions  Now we have added role base condition. if logged in user is admin than user can sync shelves.